src/constants/grid.js

```diff
@@ -18,8 +18,27 @@ const getGridExtent = () => {
 
 const EARTH_CIRCUIT = 40075000 //in meters
 
+//Tile size in degrees
+const LEVEL_0_TILE_SIZE = BASE_SIZE / 2;
+
+//size of tile in pixels
+const PIXEL_TILE_SIZE = 250;
+
+//resolutions
+//Resolution is number of meters displayed inone pixel.
+
+// Maximum resolution for level 0. Level 0 has two tiles of size 180° x 180°
+const maxResolution = EARTH_CIRCUIT / PIXEL_TILE_SIZE;
+const resolutions = []
+const levels = 25;
+for (let i = 0; i < levels; ++i) {
+	resolutions[i] = maxResolution / Math.pow(2, i + 1);
+}
+
```

Comment:
Once again, I have trouble to orient myself among your multiple PRs with the same changes. Why make another PR, if it depends on a previous one being merged first to make sense?

Reply:
You are right. Bump version should be part of the previous PR. I will rebase this PR after #2 will be merged.